### DISCUSSION TITLE
Embedding inside a View of a IOS/Android native app

### DISCUSSION THREAD

**Discussion by dominictobias** [hajimehoshi/ebiten#2439] [Q&A]
I'm wondering if anyone has ideas or resources on loading this inside a View in a native IOS or Android app?

For example with SwiftUI you are able to render low level Metal in a view: https://www.youtube.com/watch?v=H2ufvcNvVmA&list=PLlpHETvPaVAqvZMAahAhzJZlSSHceAK2M&index=1&t=1315s&ab_channel=GetIntoGameDev

**Comment by hajimehoshi** [comment]
I'm not sure this answers your question, but you can communicate from/to Go to/from Java/Objective-C by binding:

https://pkg.go.dev/golang.org/x/mobile/cmd/gobind

This way is of course available from ebitenmobile.
